docs/core/preview3/tools/csproj.md

```diff
@@ -136,34 +197,94 @@ The following example specifies the fallbacks only for the `netcoreapp1.0` targe
+
+### PackageId
+Specify a name for the resulting package. If not specified, the `pack` operation will default to using the AssemblyName or directory name as the name of the package. 
+
+### Title
+A human-friendly title of the package, typically used in UI displays as on nuget.org and the Package Manager in Visual Studio. If not specified, the package ID is used instead.
+
+### Authors
+A semicolon-separated list of packages authors, matching the profile names on nuget.org. These are displayed in the NuGet Gallery on nuget.org and are used to cross-reference packages by the same authors.
+
+### Description
+A long description of the package for UI display.
+
+### Copyright
+Copyright details for the package.
+
+### PackageRequireLicenseAcceptance
+A Boolean value specifying whether the client must prompt the consumer to accept the package license before installing the package.
+
+### PackageLicenseUrl
+An URL to the license that is applicable to the package.
+
+### PackageProjectUrl
+A URL for the package's home page, often shown in UI displays as well as nuget.org.
+
+### PackageIconUrl
+A URL for a 64x64 image with transparenty background to use as the icon for the package in UI display.
+
+### PackageReleaseNotes
+Release notes for the package.
+
+### PackageTags
+A list of tags that designates the package. The list is represented as a list of tags that are semicolon-separated. 
+
+### PackageOutputPath
+Determines the output path in which the packed package will be dropped. Default is `$(OutputPath)`. 
+
+### IncludeSymbols
+This boolean value indicates whether the package should create an additional, debug package when it packs. This package will have have a `.symbols.nupkg` extension and will put the PDB files next to the DLL files.
+
+### IncludeSource
+This value indicates whether the pack process should create a source package. The source package contains the library's source code as well as PDB files. Source files are put under the `src/ProjectName` directory in the resulting package file. 
+
+### PackageTypes
+
+
+### IsTool
+Specifies whether all output files, as specified in the Output Assemblies scenario, are copied to the tools folder instead of the lib folder. Note that this is different from a DotNetCliTool which is specified by setting the PackageType in csproj file.
+
+### RepositoryUrl
+Specifies the URL for the repository where the source code for the package resides and/or from which it is being built. 
+
+### RepositoryType
+Specifies the type of the repository. Default is "git". 
+
+### NoPackageAnalysis
+Specifies that pack should not run package analysis after building the package.
+
+### MinClientVersion
+Specifies the minimum version of the NuGet client that can install this package, enforced by nuget.exe and the Visual Studio Package Manager.
+
+### IncludeBuildOutput
+This is a boolean, which decided whether the build output assemblies should be packed into the nupkg or not.
```

Comment:
This is a boolean, which decided -> This Boolean value specifies
nupkg -> .nupkg file